### PR TITLE
Fix for Powheg+Madpsin gridpacks

### PR DESCRIPTION
**Pull request by jfernan2**
Following:

https://hypernews.cern.ch/HyperNews/CMS/get/generators/3662/1/1/1/1/1.html

Not sure if this should be the solution for other gridpacks than Powheg + Madspin

### CODE REVIEW

**Comment by perrozzi** [comment]
thanks
have you tried to check whether this has an impact on the other gridpacks?
(@kdlong @qliphy )

**Comment by efeyazgan** [comment]
Any news?

File: bin/MadGraph5_aMCatNLO/runcmsgrid_NLO.sh [hunk 1]
```diff
@@ -104,8 +104,8 @@ if [ ! -e $LHEWORKDIR/header_for_madspin.txt ]; then
 #else handle external tarball
 else
     cd $LHEWORKDIR/external_tarball
-
-    ./runcmsgrid.sh $nevtjob $rnum $ncpu
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/runcmsgrid_NLO.sh]: btw, it looks to me that if you remove nevtjob here, it is not used anywhere else, so I would suggest to remove it everywhere (cfr L71)

Comment (jfernan2) [reply bin/MadGraph5_aMCatNLO/runcmsgrid_NLO.sh]: I may be wrong, but I'm afraid it is used for pure MG5_aMC@NLO gridpacks. That's the purpouse of that variable.

Comment (perrozzi) [reply bin/MadGraph5_aMCatNLO/runcmsgrid_NLO.sh]: if I click on "view" I can't see this variable used anywhere, unless I am missing something
https://github.com/jfernan2/genproductions/blob/7eb379735966dcce8786bb43c8171fb6b7bb8a9d/bin/MadGraph5_aMCatNLO/runcmsgrid_NLO.sh

Comment (jfernan2) [reply bin/MadGraph5_aMCatNLO/runcmsgrid_NLO.sh]: It is stored on runscript.dat and then used in L96

**Pull request merged**